pyproject.toml

```diff
@@ -1,3 +1,31 @@
 [build-system]
-# PEP 518 - minimum build system requirements
-requires = ["setuptools>=61,<72", "wheel", "Cython>=0.29", "packaging", "pip>=19.0.0", "numpy", "mpi4py", "pkgconfig"]
+requires = ["setuptools>=61", "wheel", "Cython>=0.29", "packaging", "pip>=19.0.0", "numpy", "mpi4py", "pkgconfig", "setuptools-git-versioning"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pyprecice"
+dynamic = [ "version" ]
+readme = "README.md"
+description="Python language bindings for the preCICE coupling library"
+license = "LGPL-3.0"
```

Comment:
Is it LGPL-3.0 or LGPL-3.0-or-later?

Reply:
Unless we want to extend to apply the terms coming from LGPL versions > 3.0, we should stick to LGPL-3.0. @MakisH do you have an idea of what do we go for?

Reply:
Lets keep what we have now and change it later if required.